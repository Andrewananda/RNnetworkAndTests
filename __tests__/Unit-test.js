import 'react-native';
import React from 'react';

const {Utils} = require('../src/Component/Utils');

import renderer from 'react-test-renderer';

it('should check the matches and user', () => {
  const user = {
    name: 'Andrew Ananda',
    createdAt: new Date(),
  };

  expect(user).toMatchSnapshot({
    name: 'Andrew Ananda',
    createdAt: expect.any(Date),
  });
});

it('should check the matches user password', () => {
  const userPassword = {
    password: '1234',
    confirmPassword: '1234',
  };

  expect.assertions(1);

  let userData = Utils.CheckUserDetails(userPassword);
  expect(userData).toBe(1);
});
