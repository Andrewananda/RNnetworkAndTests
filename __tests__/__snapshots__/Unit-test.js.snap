// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should check the matches and user 1`] = `
Object {
  "createdAt": Any<Date>,
  "name": "Andrew Ananda",
}
`;

exports[`should check the matches user password 1`] = `
Object {
  "password": "1234",
}
`;
